README.md

```diff
@@ -1 +1,60 @@
-# leapp-report-analyzer
\ No newline at end of file
+# Leapp Report Analyzer
+
+Leapp Report Analyzer is a command-line tool for analyzing multiple **[Leapp](https://github.com/oamg/leapp)** reports and grouping systems by severity and entries.
+The tool processes JSON files containing Leapp reports and generates a JSON file and an HTML table displaying the results.
+
+By analyzing Leapp reports and providing organized output, the Leapp Report Analyzer simplifies the process of identifying and addressing potential issues in your systems.
+
+More information about Leapp:
+- [Leapp GitHub repository](https://github.com/oamg/leapp) - report schemas can be found here
+- [Leapp Documentation](https://leapp.readthedocs.io/en/latest/)
+
+## Installation
+
+Download the Python script from this repository or use:
+
+```bash
+curl -OJL https://raw.githubusercontent.com/oamg/leapp-report-analyzer/main/leapp-report-analyzer.py
+```
+
+### Requirements
+
+Script works on both Python2 and Python3, tested with:
+
+- Python 2.7
+- Python 3.3
```

Comment:
code doesn't have any version specific syntax, so maybe it would just be best to write that it works on all python 2 a 3 supported versions

Reply:
py2 is not supported anymore officially by Python community so it could be a little bit confusing. I would rather write something like:
```
- Python 2.7
- Python 3.6+
```

As there is not used effectively any Python 3.[0-5] nowadays.

Reply:
"""
Python 2.7
Python 3.2+
"""
Alright above is fine with me